dashboard/lib/dynamic_config/adapters/dynamodb_adapter.rb

```diff
@@ -0,0 +1,51 @@
+require 'aws-sdk'
+
+class DynamoDBAdapter
+  def initialize(table_name)
+    @table_name = table_name
+    @client = Aws::DynamoDB::Client.new
+  end
+
+  def get(key)
+    resp = @client.get_item({
+      table_name: @table_name,
+      key: {
+        'data-key' => JSON.dump(key)
+      }
+    })
+    return nil if resp.item.nil?
+    JSON.load(resp.item['data-value'])
+  end
+
+  def set(key, value)
+    @client.put_item({
+      table_name: @table_name,
+      item: {
+        'data-key' => JSON.dump(key),
```

Comment:
Oh wow, that's ... interesting. All other json libraries have used have raised a JSON error if something is not json encodable. Seems weird that they just use the object id.  I'll update it to raise an exception.